src/gui/qgslayermetadatasourceselectprovider.cpp

```diff
@@ -34,6 +34,11 @@ QString QgsLayerMetadataSourceSelectProvider::text() const
   return QObject::tr( "Metadata Search" );
 }
 
+QString QgsLayerMetadataSourceSelectProvider::toolTip() const
+{
+  return QObject::tr( "Search and add layer metadata" );
```

Comment:
Maybe:

```suggestion
  return QObject::tr( "Find layers by metadata" );
```

Reply:
@nyalldawson You are probably right... the only reference I have is unfortunately the misleading docs.